OutOfSchool/OutOfSchool.WebApi.Tests/Services/Database/WorkshopServicesCombinerDBTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using AutoMapper;
+using Moq;
+using NUnit.Framework;
+using OutOfSchool.ElasticsearchData;
+using OutOfSchool.ElasticsearchData.Models;
+using OutOfSchool.Services.Enums;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Repository;
+using OutOfSchool.Tests.Common;
+using OutOfSchool.Tests.Common.TestDataGenerators;
+using OutOfSchool.WebApi.Models.Workshops;
+using OutOfSchool.WebApi.Services;
+using OutOfSchool.WebApi.Services.Strategies.Interfaces;
+using OutOfSchool.WebApi.Util;
+using OutOfSchool.WebApi.Util.Mapping;
+
+namespace OutOfSchool.WebApi.Tests.Services.Database;
+
+[TestFixture]
+public class WorkshopServicesCombinerDBTests
+{
+    private Mock<IWorkshopService> workshopService;
+    private Mock<IApplicationRepository> applicationRepository;
+    private Mock<IEntityRepositorySoftDeleted<long, Favorite>> favoriteRepository;
+    private Mock<IElasticsearchSynchronizationService> elasticsearchSynchronizationService;
```

Comment:
(Excessive code) You could get rid of these 3 fields, as they are used only in `SetUp()`, thus could be made a local variables there.

Reply:
Of course. Sorry for my inattention. I fixed it.